src/utils/hooks/useYoutubeVideos.js

```diff
@@ -1,23 +1,27 @@
 import { useState, useEffect } from 'react';
+import { useGlobalContext } from '../../providers/GlobalProvider/GlobalProvider';
 import { apiGet } from '../services/service';
 
 export function useYoutubeVideos() {
-  const [videos, setVideos] = useState([]);
   const [search, setSearch] = useState('');
+  const { setVideosCall } = useGlobalContext();
 
   const getVideos = async (newSearch = '') => {
-    const response = await apiGet('/search#video', {
+    const {
+      data: { items },
+    } = await apiGet('/search#video', {
       params: {
         q: newSearch,
         type: 'video',
       },
     });
-    setVideos(response.data);
+    setVideosCall(items);
   };
 
   useEffect(() => {
     getVideos(search);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
same comment about removing this disabling rule comment

this will ask you to add `getVideos` to the dependency array and if that's behaving in a way that is making this run more times than necessary then we might need to add a `useCallback` to `getVideos` definition.

have not tested this but something like

```js
  const getVideos = useCallback((newSearch = '') => {
   const getVideosAsync = async () => {
    const response = await apiGet('/search#video', {
    const {
      data: { items },
    } = await apiGet('/search#video', {
      params: {
        q: newSearch,
        type: 'video',
      },
    });
    setVideos(response.data);
    setVideosCall(items);
    })
   
    getVideosAsync();
   }});
```